tests/scripts/all.sh

```diff
@@ -3856,13 +3879,36 @@ component_test_full_block_cipher_psa_dispatch () {
 
     helper_libtestdriver1_make_main "$loc_accel_list"
 
+    # Make sure disabled components were not re-enabled by accident (additive
+    # config)
+    not grep mbedtls_aes_ library/aes.o
+    not grep mbedtls_aria_ library/aria.o
+    not grep mbedtls_camellia_ library/camellia.o
+
     # Run the tests
     # -------------
 
     msg "test: full + PSA dispatch in block_cipher"
     make test
 }
 
+# This is the reference component of component_test_full_block_cipher_psa_dispatch
+component_test_full_block_cipher_legacy_dispatch () {
+    msg "build: full + legacy dispatch in block_cipher"
+
+    common_block_cipher_dispatch 0
+
+    # Disable cipher modes other than ECB as in the accelerated component. ECB
+    # does not have a configuration symbol and it's automatically enabled as
+    # long as underlying key types are.
+    scripts/config.py unset-all MBEDTLS_CIPHER_MODE
```

Comment:
Same suggestion: does putting this in the common function reduce the number of things we have to ignore in `analyze_outcomes.py`?